modules/oc-mirror-dry-run.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-dry-run_{context}"]
+= Performing a dry run
+
+You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored, as well as any images that would be pruned from the mirror registry. It also allows you to catch any errors with your image set configuration early.
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+* You have created the image set configuration file.
+
+.Procedure
+
+. Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  file://<path_to_output_directory>         \ <2>
+  --dry-run                                   <3>
+----
+<1> Pass in the image set configuration file that was created. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Specify the target directory where you want to output the dry run artifacts. The target directory path must start with `file://`.
+<3> Use the `--dry-run` flag to generate the dry run artifacts and not an actual image set file.
++
+.Example output
+[source,terminal]
+----
+Creating directory: <path_to_output_directory>/oc-mirror-workspace/src/publish
+Creating directory: <path_to_output_directory>/oc-mirror-workspace/src/v2
+Creating directory: <path_to_output_directory>/oc-mirror-workspace/src/charts
+Creating directory: <path_to_output_directory>/oc-mirror-workspace/src/release-signatures
+No metadata detected, creating new workspace
+wrote mirroring manifests to <path_to_output_directory>/oc-mirror-workspace/operators.1658342351/manifests-redhat-operator-index
+
+...
+
+info: Planning completed in 31.48s
+info: Dry run complete
+Writing image mapping to <path_to_output_directory>/oc-mirror-workspace/mapping.txt
+----
+
+. Navigate into the workspace directory that was generated:
++
+[source,terminal]
+----
+$ cd <path_to_output_directory>/oc-mirror-workspace/
+----
+
+. Review the `mapping.txt` file that was generated.
++
+This file contains a list of all images that would be mirrored.
+
+. Review the `pruning-plan.json` file that was generated.
++
+This file contains a list of all images that would be pruned from the mirror registry when the image set is published.
++
+[NOTE]
+====
+The `pruning-plan.json` file is only generated if there are images to be pruned.
```

Comment:
We could include here that the `pruning-plan.json` would only be generated with a `mirror-to-mirror` to `disk-to-mirror` workflow.